Fix implicit function declaration (error as of Xcode 12):
secmem_free() is declared in "memory.h" rather than <memory.h>
(the <memory.h> system header merely inludes <string.h>)
https://trac.macports.org/ticket/61421

Upstream-Status: Submitted (https://github.com/GPGTools/pinentry-mac/pull/8)

diff --git a/Source/pinentry-0.9.4/pinentry/pinentry-curses.c b/Source/pinentry-0.9.4/pinentry/pinentry-curses.c
index fc27d81..4279aff 100644
--- a/Source/pinentry-0.9.4/pinentry/pinentry-curses.c
+++ b/Source/pinentry-0.9.4/pinentry/pinentry-curses.c
@@ -44,7 +44,7 @@
 #include <utime.h>
 #endif /*HAVE_UTIME_H*/
 
-#include <memory.h>
+#include "memory.h"
 
 #ifdef HAVE_WCHAR_H
 #include <wchar.h>
